### DISCUSSION TITLE
qemu + ipv6 promblem

### DISCUSSION THREAD

**Discussion by sprootik** [ipxe/ipxe#1191] [General]
Hello.
I made ipxe with ipv6 support and debugging. For the test, I try to run on qemu  (UEFI) and get an error. 
It seems to me that during normal operation the ipxe should load and receive an IP address
![ipxe_problem](https://github.com/ipxe/ipxe/assets/38777214/244b6935-b649-456a-bcf8-b7734e0a12df)

dump in attach
[dump.tar.gz](https://github.com/ipxe/ipxe/files/14927020/dump.tar.gz)

**Comment by NiKiZe** [comment]
Are you using autoexec.ipxe or any embedded script?
Which local modifications do you have if any?

**Reply by sprootik** [reply]
No, I don't use embedded.
I planned to send a link for ipxe config in the dhcp packet.

I just enable dhcpv6 in config/general.h and log in config/console.h

**Reply by NiKiZe** [reply]
Have you tried with a valid link? (Or at least a valid formatted one)

Every iPXE error message has the ipxe.org URL as you can see, please read that linked page as well.

If you still have issues, please post a text representation of your dhcpoffer tcpdump -s 0 -v should be good.

Also try without modifications first.
For example ipv6 is already enabled by default in EFI builds https://github.com/ipxe/ipxe/blob/59f27d69358efc919b50760f3d6dac0b637b5488/src/config/defaults/efi.h#L30

**Reply by sprootik** [reply]
Rebuilt, same result. I attached the dump in the first message, if you need the next one, I can attach it.

I don't understand what this URL is. 
With a regular pxe loader I teach IP and load ipxe (ipxe.efi).  after this, ipxe should again receive the address via DHCP. Instead it swears at some url..

**Reply by NiKiZe** [reply]
iPXE uses URLs
But does iPXE really stop at this point? What does a clean, no source changes, give for output? And what does the relevant DHCP packages look like?

I do not believe that this is a crash or error in iPXE, if it is then it is not related to this message.

**Reply by sprootik** [reply]
Yes, ipxe stop at this point.
A little bit later I'll take a dump with ipxe.efi without any changes

**Reply by NiKiZe** [reply]
Then we need steps to reproduce, no dump will help me help you 

Version of qemu, qemu command line, and again, dump of your DHCP packages on the network.

**Comment by mcb30** [comment]
I have been able to reproduce the problem and have created issue #1192 to track it.
